libflux/src/flux/semantic/flatbuffers/semantic_generated.rs

```diff
@@ -2998,6 +3004,30 @@ pub mod fbsemantic {
                 None
             }
         }
+
+        #[inline]
+        #[allow(non_snake_case)]
```

Comment:
Is this lint rule part of the generated code, or did it get added manually?

Reply:
Yeah, everything in this file is generated.